pyls/plugins/flake8_lint.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2019 Palantir Technologies, Inc.
+"""Linter pluging for flake8"""
+import logging
+from flake8.api import legacy as flake8
+from pyls import hookimpl, lsp
+
+log = logging.getLogger(__name__)
+
+
+@hookimpl
+def pyls_lint(config, document):
```

Comment:
Should we default this plugin to disabled, and when it is enabled disable pep8 and pyflakes? I think at the moment we’d end up with duplicate diagnostics which could be annoying

Reply:
yeah, they shouldn't be running at the same time, and disabling flake8 by default is a good idea so current users don't feel the change but can enable linting directly via flake8 if they want to. But flake8_config have to be made loading config into flake8 plugin as well from now on.